### ISSUE TITLE
Consider more accessible colour contrast schemes

### ISSUE THREAD

**Issue by BBlackwo** [gureckis/slidev-theme-neversink#10]
First off, thanks for the excellent slidev theme! It's my favourite I've seen so far.

I noticed that many of the [colour schemes](https://gureckis.github.io/slidev-theme-neversink/colors.html) do not pass [accessible colour contrast guidelines](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). I used [WAVE](https://wave.webaim.org/) to generate the list below. The colours with a `x` don't meet the minumim contrast ratio of 3:1.

| Light Schemes | Regular Schemes |
|--------------|-----------|
| <img width="437" alt="image" src="https://github.com/user-attachments/assets/d8c38321-3ab7-46e5-b481-75c81a3bc932"> | <img width="454" alt="image" src="https://github.com/user-attachments/assets/cc256165-0aff-4379-a85e-835c1fb97bc6"> |

It's worth considering updating the colour schemes to be accessible as it will make the slides easier to read. Particularly for people with low vision or when using projectors which have poor colour contrast.

There are a variety of tools which can help create accessible colour palettes such as https://venngage.com/tools/accessible-color-palette-generator

**Comment by BBlackwo** [comment]
Or a simpler fix would be to make the colours that aren't accesible slightly darker when generating the colour schemes.

I tried it out the following in https://github.com/BBlackwo/slidev-theme-neversink/commit/0ed01d50bec8d929cdbc55ce3f9f3f82b19124c4 and I think it makes them all accessible:
1. Left the ones that are already accessible
2. Bumped up orange, green, emerald, teal, cyan, sky, and fuchsia by 100
3. Bumped up amber, yellow, and lime by 200

| Light Schemes | Regular Schemes |
|--------------|-----------|
| <img width="464" alt="Screenshot 2024-07-23 at 9 05 05 PM" src="https://github.com/user-attachments/assets/24a0c1c1-b7dd-41b9-8096-3638bd27ad9a"> | <img width="462" alt="Screenshot 2024-07-23 at 9 04 52 PM" src="https://github.com/user-attachments/assets/140d4c88-7f9b-4652-9f28-f9b8cc451b56"> |

**Comment by gureckis** [comment]
yeah this is a great idea and agree the approach worked well for many of the colors.  i am a bit sad about amber-light, lime-light as they were my favorite colors.  and they lost a bit of the monochrome feel.  perhaps theres a way to make a slight change to those two.  but agree and understand why this is a good idea.  my set was arbitrary and only used combinations available in the unocss tailwind preset.  feel free to make a pr!

**Comment by gureckis** [comment]
closing this issue as stale for now, but if someone what to work on it, welcome to reopen!
